appendixes/mysite/tests/behat/features/bootstrap/Context/FeatureContext.php

```diff
@@ -0,0 +1,121 @@
+ */
+class FeatureContext extends SilverStripeContext {
+    
+    /**
+     * @var FixtureFactory
+     */
+    protected $fixtureFactory;
+
+    /**
+     * Initializes context.
+     * Every scenario gets it's own context object.
+     *
+     * @param array $parameters context parameters (set them up through behat.yml)
+     */
+    public function __construct(array $parameters) {
+        parent::__construct($parameters);
+
+        $this->useContext('BasicContext', new BasicContext($parameters));
+        $this->useContext('LoginContext', new LoginContext($parameters));
+        $this->useContext('CmsFormsContext', new CmsFormsContext($parameters));
+        $this->useContext('CmsUiContext', new CmsUiContext($parameters));
+
+        $fixtureContext = new FixtureContext($parameters);
+        $fixtureContext->setFixtureFactory($this->getFixtureFactory());
+        $this->useContext('FixtureContext', $fixtureContext);
+
+        // Use blueprints to set user name from identifier
+        $factory = $fixtureContext->getFixtureFactory();
+        $blueprint = \Injector::inst()->create('FixtureBlueprint', 'Member');
+        $blueprint->addCallback('beforeCreate', function($identifier, &$data, &$fixtures) {
+            if(!isset($data['FirstName'])) $data['FirstName'] = $identifier;
+        });
+        $factory->define('Member', $blueprint);
+
+        // Auto-publish pages
+        if (class_exists('SiteTree')) {
+            foreach(\ClassInfo::subclassesFor('SiteTree') as $id => $class) {
+                $blueprint = \Injector::inst()->create('FixtureBlueprint', $class);
+                $blueprint->addCallback('afterCreate', function($obj, $identifier, &$data, &$fixtures) {
+                    $obj->publish('Stage', 'Live');
+                });
+                $factory->define($class, $blueprint);
+            }
+        }
+    }
+
+    public function setMinkParameters(array $parameters) {
+        parent::setMinkParameters($parameters);
+        
+        if(isset($parameters['files_path'])) {
+            $this->getSubcontext('FixtureContext')->setFilesPath($parameters['files_path']);    
+        }
+    }
+
+    /**
+     * @return FixtureFactory
+     */
+    public function getFixtureFactory() {
+        if(!$this->fixtureFactory) {
+            $this->fixtureFactory = \Injector::inst()->create('BehatFixtureFactory');
+        }
+
+        return $this->fixtureFactory;
+    }
+
+    public function setFixtureFactory(FixtureFactory $factory) {
+        $this->fixtureFactory = $factory;
+    }
+
+    /**
+     * "Shares" the database with web requests, see
+     * {@link MeridianFakeManagerControllerExtension}
+     */
+    public function getTestSessionState() {
+        return array_merge(
+            parent::getTestSessionState(),
+            array(
+                'useFakeManager' => true,
+                'importDatabasePath' => BASE_PATH .'/mysite/tests/fixtures/SS-sample.sql',
```

Comment:
You're assuming that every Behat usage will rely on a default SQL file. This adds additional overhead to each scenario execution, since we need to run `dev/build` on top of that SQL import anyway to avoid schema mismatches between the SQL file and the actual schema defined in code. Hiding fixtures away in SQL files is kind of an anti-pattern for testing, since it creates "magic state" that just exists in test executions, rather than explicitly creating fixtures through step definitions. For these two reasons, I think we should remove the SQL file from here, and simply document this as an advanced use case in the README.